src/python/grpcio/tests/unit/test_common.py

```diff
@@ -61,6 +61,10 @@ def metadata_transmitted(original_metadata, transmitted_metadata):
   original = collections.defaultdict(list)
   for key_value_pair in original_metadata:
     key, value = tuple(key_value_pair)
+    if not isinstance(key, bytes):
```

Comment:
This feels like something that should be done with a call to `six` but I'm not sure what. Have you looked and not found the right thing?